.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
No all IDEs support this. Let's remove this file and implement in a future task. Can be made part of the build (ie. checkstyle).

Reply:
Its there for the IDEs which support this :)